bin/hdfcoinc/pycbc_coinc_statmap

```diff
@@ -56,12 +56,43 @@ parser.add_argument('--max-hierarchical-removal', type=int, default=0,
                     help='Maximum amount of hierarchical removals to carry '
                          'out when hierarchical removal is desired. Choose '
                          '-1 for continuous hiearchical removal until no '
-                         'foreground triggers are louder than the inclusive '
-                         'background. Choose 0 to do no hierarchical '
-                         'removals.  Choose 1 to do at most 1 hierarchical '
-                         'removal, and so on. [default=0]')
```

Comment:
Add help that says ``--hierarchical-removal-against`` is required if this argument is given.